io.catenax.shared.shopfloor_information_types/RELEASE_NOTES.md

```diff
@@ -0,0 +1,7 @@
+# Changelog
+
+All notable changes to this model will be documented in this file. 
+
+## [1.0.0]
```

Comment:
don't forget to add the date of the MS3 approval to the RELEASE_NOTES once it is MS3 approved